storage/kvstore/kv_config/global/mbed_lib.json

```diff
@@ -2,7 +2,7 @@
 "name": "storage",
     "config": {
         "storage_type": {
-            "help": "Options are TDB_INTERNAL, TDB_EXTERNAL, TDB_EXTERNAL_NO_RBP, FILESYSTEM, FILESYSTEM_NO_RBP or default. If default, the storage type will be chosen according to the component defined in targets.json",
+            "help": "Options are TDB_INTERNAL, TDB_EXTERNAL, FILESYSTEM or default. If default, the storage type will be chosen according to the component defined in targets.json",
```

Comment:
@0xc0170 
If we don't want a breaking change (removal of `TDB_EXTERNAL_NO_RBP` and `FILESYSTEM_NO_RBP`) for now, the most we can do is say

    `TDB_EXTERNAL_NO_RBP` and `FILESYSTEM_NO_RBP` do not work and are thus deprecated

in this help text and above functions that define them. We may not benefit from compile-time deprecation warnings due to preprocessor being unable to compare macros with strings, but I don't have any best ideas.